lib/shopify_cli/constants.rb

```diff
@@ -58,6 +58,18 @@ module EnvironmentVariables
       MONORAIL_REAL_EVENTS = "MONORAIL_REAL_EVENTS"
     end
 
+    module SupportedVersions
+      module Ruby
+        FROM = "2.6.6"
+        TO = "3.0.2"
```

Comment:
Sorry for joining the party late, but I just saw the version of Ruby we're recommending.

Homebrew users currently are on 3.0.3 so they'll all be warned.

Can the check be a bit less strict, maybe we say `< 3.1` instead of `<= 3.0.2`?

Reply:
Good point! Looking at the code, it seems that the `FROM` version (2.6.6) is accepted, but the `TO` (3.0.2) is not...

https://github.com/Shopify/shopify-cli/blob/328409388db16ed01e929b3fd8b976efa88d003d/lib/shopify_cli/command.rb#L113

So I think the best way would be to change that to `<=` and use "3.0" as the `TO` instead.

Reply:
It was finally fixed here: https://github.com/Shopify/shopify-cli/pull/2022